alias

```diff
@@ -49,6 +49,22 @@ LAST_AMN_AND_YEAR="${LAST_AMN}${THIS_YEAR}" # "apr2044"
 if [[ "$LAST_AMN" == "dec" ]]; then
     LAST_AMN_AND_YEAR="${LAST_AMN}${LAST_YEAR}"
 fi
+
+THIS_MONTH_DIGIT=$(date +'%m')
```

Comment:
You could just squash all commits into one, then remove those aliases and ``commit --amend`` & force push. That'll be clean.

Reply:
Will do, after making some of the other changes you recommend!

Reply:
I've finished making the changes for now. But for some reason Github wont let me take this back out of draft mode... I found this help page: https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request
Seems to suggest that I need access to your repo to do it.... despite it being my pull request... Weird.
Still, hopefully you'll be able to just press the button then merge it in.